code/initParameterIdentificationForOSPSuiteExport.m

```diff
@@ -1,18 +1,18 @@
-function PI=initParameterIdentificationForOSPSuiteExport(PI_xml,simulationList)
-% INITPARAMETERIDENTIFICATIONFOROSPSUITEEXPORT initialize the exported parameter identification
+function PI=initParameterIdentificationForSBSuiteExport(PI_xml,simulationList)
+% INITPARAMETERIDENTIFICATIONFORSBSUITEEXPORT initialize the exported parameter identification
 % 
```

Comment:
The whole file initParameterIdentificationForOSPSuiteExport.m seems to be overwritten with the old version??